### PR TITLE
Add Anomaly Detection to 2.8.0 manifests

### PR DESCRIPTION
**Pull request by kaituo**
### Description
Add AD frontend and backend to 2.8.0 manifest

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/2.8.0/opensearch-2.8.0.yml [hunk 1]
```diff
@@ -137,3 +137,12 @@ components:
       - windows
     checks:
       - gradle:properties:version
+  - name: anomaly-detection
+    repository: https://github.com/opensearch-project/anomaly-detection.git
+    ref: tags/2.8.0.0
```
Comment (gaiksaya) [inline manifests/2.8.0/opensearch-2.8.0.yml:142]: should be branch not tag!

File: manifests/2.8.0/opensearch-dashboards-2.8.0.yml [hunk 2]
```diff
@@ -25,3 +25,6 @@ components:
   - name: indexManagementDashboards
     repository: https://github.com/opensearch-project/index-management-dashboards-plugin.git
     ref: '2.8'
+  - name: anomalyDetectionDashboards
+    repository: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin
+    ref: tags/2.8.0.0
```
Comment (gaiksaya) [inline manifests/2.8.0/opensearch-dashboards-2.8.0.yml:30]: should be branch not tag

File: manifests/2.8.0/opensearch-2.8.0.yml (hunk 1 above)
Comment (kaituo) [reply manifests/2.8.0/opensearch-2.8.0.yml]: changed

File: manifests/2.8.0/opensearch-dashboards-2.8.0.yml (hunk 2 above)
Comment (kaituo) [reply manifests/2.8.0/opensearch-dashboards-2.8.0.yml]: changed

**Review by gaiksaya — APPROVED** [review]

**Comment by peterzhuamazon** [comment]
This is duplicated with @rishabh6788 PR:
* https://github.com/opensearch-project/opensearch-build/pull/3572

Also 2.8 branch still not cut so will keep failing.
Closing.

**Pull request closed without merging**